docker/nginx/nginx.conf

```
@@ -33,6 +33,23 @@ http {
 
   server {
     listen 80;
+    return 301 https://$TARGET_HOST$request_uri;
```

$TARGET_HOST refers to the target vor container inside the docker network. I think what you're looking for is something like this:
```suggestion
    return 301 https://$host$request_uri;
```
As far as I understand it, this one redirects the user to the same exact URL, but with https instead of HTTP.